charmcraft/templates/init-django-framework/pyproject.toml.j2

```diff
@@ -0,0 +1,41 @@
+# Testing tools configuration
+[tool.coverage.run]
+branch = true
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.pytest.ini_options]
+minversion = "6.0"
+log_cli_level = "INFO"
+
+# Linting tools configuration
+[tool.ruff]
+line-length = 99
+lint.select = ["E", "W", "F", "C", "N", "D", "I001"]
+lint.extend-ignore = [
+    "D105",
+    "D107",
+    "D203",
+    "D204",
+    "D213",
+    "D215",
+    "D400",
+    "D404",
+    "D406",
+    "D407",
+    "D408",
+    "D409",
+    "D413",
```

Comment:
maybe add a comment for what each these are as this is a template

Reply:
These came across from the other templates, so I'd have to research them and probably update the other templates too. I've created https://github.com/canonical/charmcraft/issues/1964 for this.